web-common/src/features/connectors/connectors-utils.ts

```diff
@@ -17,6 +22,10 @@ export function makeFullyQualifiedTableName(
       return `${database}.${databaseSchema}.${table}`;
     case "pinot":
       return table;
+    case "starrocks":
```

Comment:
these qualified table names or frontend integration related changes should be implemented in the frontend scope